kafkajobs/config.go

```diff
@@ -30,6 +35,76 @@ func (c *config) InitDefault() ([]kgo.Opt, error) {
 	opts = append(opts, kgo.MaxVersions(kversion.Stable()))
 	opts = append(opts, kgo.RetryTimeout(time.Minute*5))
 
+	if c.TLS != nil && c.TLS.Enabled {
```

Comment:
I'm still thinking, that we don't need this option (`c.TLS.Enabled`). RR supports default env bash syntax, like `${foo:-default}`, if `foo` env is set, use it, no - use `default` 
So, TLS options might be easily controlled via envs. And since we don't have this `enabled` option in any other config, this would be confusing, why do we have it only here.
Do not need to expose `InsecureSkipVerify` it should be always set to `false`.

Reply:
fixed